scripts/install/opt/seagate/cortx/motr/bin/motr_setup.py

```diff
@@ -25,7 +25,8 @@
 import traceback
```

Comment:
I am sorry, I suggested to change the name to "motr_setup.py".

But this script is used by provisioner. Before merging this PR, we have to get approval from integration group.

Reply:
@atulsdeshmukh2312  @yeshpal-jain-seagate : Can you check the impact of this change ?